sd_notify.gemspec

```diff
@@ -9,4 +11,10 @@ Gem::Specification.new do |s|
   s.files       = ["lib/sd_notify.rb", "LICENSE", "README.md", "CHANGELOG.md"]
   s.homepage    = "https://github.com/agis/ruby-sdnotify"
   s.license     = "MIT"
+
+  s.required_ruby_version = ">= 2.3.0"
```

Comment:
No real reason. In `teardown` in tests I used `&.` operator which is available only since 2.3, but I can definitely revert back to `@socket.close if @socket` syntax.

Reply:
Yeah I don't think it's worth it. Let's support 2.3 and earlier versions.